### PR TITLE
Update composer dependencies

### PR DESCRIPTION
**Pull request by ruflin**

### CODE REVIEW

**Comment by ruflin** [comment]
@p365labs Would be great.

If you think this one is ok to go, can you "Approve" the PR?

**Comment by p365labs** [comment]
@ruflin  for me it's ok can I merge it ?

**Comment by ruflin** [comment]
@p365labs Just merged it.

File: env/elastica/composer.json [hunk 1]
```diff
@@ -13,14 +13,14 @@
...
-        "phpunit/phpunit":"~5.6",
-        "sebastian/phpcpd":"~2.0",
-        "squizlabs/php_codesniffer":"~2.5"
+        "phpdocumentor/phpdocumentor": "^2.9",
+        "fabpot/php-cs-fixer": "1.11.*",
+        "mayflower/php-codebrowser": "~1.1",
+        "pdepend/pdepend": "~2.2",
+        "phploc/phploc": "~2.1",
+        "phpmd/phpmd": "~2.3",
+        "phpunit/phpunit": "~5.6",
```
[the remark this answers is not in the document]
Comment (ruflin) [reply env/elastica/composer.json]: Yes, but separate PR. I do not want to see what the affect on each single one is.

**Pull request merged**